assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -129,10 +126,4 @@ const mapStateToProps = state => ({
   lang: state.i18n.locale
 });
 
-const withData = graphql(TabsConditionQuery, {
-  props: ({ data }) => ({
-    ...data
-  })
-});
-
-export default compose(connect(mapStateToProps), withData, manageErrorAndLoading({ displayLoader: false }))(Footer);
```

Comment:
I think you have now a regression. `hasLegalNotice,	hasLegalNotice, hasTermsAndConditions, hasCookiesPolicy, hasPrivacyPolicy, hasUserGuidelines` are all undefined now.

Reply:
True, after removing the code in main we forgot to reput what is needed in the footer